src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.Gui.Gui;
+import duke.component.DukeException;
+import duke.component.Storage;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+
+public abstract class Command {
+
+    /**
+     * abstract method to be implemented by child classes.
```

Comment:
Can reconsider phrasing to a description of what code does even though it would be general in this case